TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/models/migration.py

```diff
@@ -0,0 +1,110 @@
+from console_link.logic.osi_migration import OSIMigrationLogic, OpenSearchIngestionMigrationProps
+from console_link.models.cluster import Cluster
+from typing import Dict
+from enum import Enum
+from cerberus import Validator
+
+OSI_SCHEMA = {
```

Comment:
Nit: `regex` can be used to check for valid items.  This could be useful if we see patterns of errors that are do to bad inputs

Reply:
Not sure i was grokking this. Is the suggestion to use regex patterns somewhere to validate specific fields?

Reply:
Ah, like regexes to validate that these look like arns? I like that for down the road, I think we shoudl think about exactly how we want to organize these validation schemas.

Reply:
I wouldn't change what you have for this suggestion, just letting you know that you could add a regex to validate the schema for more than a 'string', say if we are pumping data from the config directly into cdk or other templates and its important that a CIDR range is actually matches a schema.

Note; your favorite LLM could whip these up pretty quickly - however, this is only useful IF we are seeing validation errors that we want to catch here.